docs/_documentations/installlocally.md

```diff
@@ -20,9 +20,8 @@ Codewind can be installed locally into Eclipse or into Visual Studio Code (VS Co
 * Linux or macOS, on x86_64 architecture only
 * [Visual Studio Code](https://code.visualstudio.com/) or [Eclipse IDE for Java EE Developers](https://www.eclipse.org/downloads/packages/release/)
 * [Git](https://git-scm.com/)
-* [Docker Desktop for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac) **Version 17.06 minimum**
-* [Docker Compose](https://docs.docker.com/compose/install/)
-* On Linux, follow the post-installation steps to [run Docker as a non-root user](https://docs.docker.com/engine/installation/linux/linux-postinstall/).
+* On macOS, [Docker Desktop for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac) **Version 17.06 minimum**
+* On Linux, [Docker Compose](https://docs.docker.com/compose/install/), and then follow the post-installation steps to [run Docker as a non-root user](https://docs.docker.com/engine/installation/linux/linux-postinstall/).
```

Comment:
Although the linked page states it (below), should we mention that Docker Compose requires Docker Engine as a prerequisite?

"On Linux systems, first install the Docker for your OS as described on the Get Docker page, then come back here for instructions on installing Compose on Linux systems."

Reply:
@makandre I think I'd prefer to let the linked page describe that bit. I tried working it into the `On Linux...` paragraph above but it gets quite clunky, and loses the focus of installing Docker Compose and then running Docker as non-root.